spec/models/question_spec.rb

```diff
@@ -54,12 +52,17 @@
   describe "#generate_answers" do
     let(:question) { Fabricate(:question) }
 
-    it "generates 4 answers total" do
+    it "generates 4 answers from mix" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.